nipype/pipeline/engine/nodes.py

```diff
@@ -1156,8 +1037,9 @@ def _make_nodes(self, cwd=None):
                         base_dir=op.join(cwd, 'mapflow'),
                         name=nodename)
             node.plugin_args = self.plugin_args
-            node._interface.inputs.trait_set(
+            node.interface.inputs.trait_set(
                 **deepcopy(self._interface.inputs.get()))
+            node.interface.resource_monitor = self._interface.resource_monitor
```

Comment:
`self.interface.resource_monior`

Reply:
I would opt for `self._interface`. I've made sure no `self.interface` remain.